test/tizen_capi/unittest_tizen_capi.cpp

```diff
@@ -50,7 +50,7 @@ TEST(nntrainer_nnmodel_construct_deconstruct, nntrainer_02_n) {
  * @brief Neural Network Model Construct wit Configuration File Test
  * @return 0 success, -EINVAL if failed.
```

Comment:
Is it right to write the return value here?

Reply:
removed.